modules/farm_fd2_examples/src/entrypoints/submit_reset_buttons/App.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <h3>SubmitResetButtons Example</h3>
+  <p data-cy="description">
+    The SubmitResetButtons is a component for the Submit and Reset buttons used
+    in forms
+  </p>
+
+  <hr />
+  <SubmitResetButtons
+    id="submit-reset-buttons"
+    data-cy="submit-reset-buttons"
+    v-model:enableReset="this.enableReset"
+    v-model:enableSubmit="this.enableSubmit"
+    v-on:submit="this.submitEnabled = $event"
+    v-on:reset="this.resetEnabled = $event"
+    v-on:ready="createdCount++"
+  />
+  <hr />
+
+  <h5>Component Props:</h5>
+  <table class="example-table">
+    <thead>
+      <th>Prop</th>
+      <th>Control</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td>enableReset</td>
+        <td>
+          <BFormCheckbox
+            id="enable-reset-checkbox"
+            data-cy="enable-reset-checkbox"
+            switch
+            v-model="this.enableReset"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>enableSubmit</td>
+        <td>
+          <BFormCheckbox
+            id="enable-submit-checkbox"
+            data-cy="enable-submit-checkbox"
+            switch
+            v-model="this.enableSubmit"
+          />
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h5>Component Event Payloads</h5>
+  <table class="example-table">
+    <thead>
+      <th>Event</th>
+      <th>Payload</th>
```

Comment:
I feel like displaying `true` here as the payload value is a little misleading since there isn't actually a payload for these events.  Maybe instead, you can keep a click count for each button and increment it each time there is a click?  Then change the column header here here to be "Clicked".